lib/paratii.eth.vids.js

```diff
@@ -40,6 +40,7 @@ export class ParatiiEthVids {
       id: joi.string(),
       owner: joi.string().required(),
       price: joi.number().default(0),
+      author: joi.string(),
```

Comment:
shortest PR ever :-)
it should be an optional value, though.
while you are at it, can you also privde a `duration.string().empty()`?